share/getters/shrex_test.go

```diff
@@ -88,23 +89,56 @@ func TestShrexGetter(t *testing.T) {
 		require.ErrorIs(t, err, share.ErrNotFound)
 	})
 
-	t.Run("ND_namespace_not_found", func(t *testing.T) {
+	t.Run("ND_namespace_not_included", func(t *testing.T) {
 		ctx, cancel := context.WithTimeout(ctx, time.Second)
 		t.Cleanup(cancel)
 
 		// generate test data
-		eds, dah, nID := generateTestEDS(t)
+		eds, dah, maxNID := generateTestEDS(t)
 		require.NoError(t, edsStore.Put(ctx, dah.Hash(), eds))
 		peerManager.Validate(ctx, srvHost.ID(), shrexsub.Notification{
 			DataHash: dah.Hash(),
 			Height:   1,
 		})
 
 		// corrupt NID
-		nID[4]++
+		nID := make([]byte, ipld.NamespaceSize)
+		copy(nID, maxNID)
+		nID[ipld.NamespaceSize-1]-- // pray for last byte to not be 0x00
```

Comment:
LOL lets just do a check?

Reply:
I would rather wait for utility func, since it is not easy to sub 1 from multibyte int